### PR TITLE
 🐛 [Fix] learning rate schedule and momentum value

### LINKED ISSUE #122: Issues within the learning rate schedule and optimizer initialization
## Describe the bug
There are three parameter groups defined in yolo/utils/model_utils.py/create_optimizer. One for biases, one for batch_norm weights and one for the convolutional weights. In the stable implementation, the learning rate for each of these groups is the same during training, they only differ in their weight_decay (only the conv weights have it), however, in the current implementation the bias learning rate starts hight during warmup, then quickly converges towards zero right after the warmup epochs, basically freezing the bias values after a couple of epochs! This issue stems most likely from Line 79 in yolo/utils/model_utils.py/create_optimizer:
`    optimizer.max_lr = [0.1, 0, 0]`, where the max_lr is initialized differently for the bias parameters group.

An additional problem is, that the momentum values for the optimizer are hardcoded to be 0.8 instead of using the value found in train.yaml, which is 0.937 in the following code snippet (Lines 55-57):
```
        {"params": bias_params, "momentum": 0.8, "weight_decay": 0},
        {"params": conv_params, "momentum": 0.8},
        {"params": norm_params, "momentum": 0.8, "weight_decay": 0},
```

## To Reproduce
1. Train a network with tensorboard enabled.
2. Observe the learning rate of the parameter groups.

## Expected behavior
The bias learning rate not to converge towards zero early.

## Screenshots
Learning rate schedule for the three parameter group in the current implementation:
learning_rates

Learning rate schedule for the same parameter groups in the "stable" implementation:
lr-stable

## Proposed solution
 - Set Line 79 to `    optimizer.max_lr = [0, 0, 0]` for the learning rates to be identical.
 - Remove the fix momentum values from Line 55-57, in which case the optimizer is initialized with value provided in train.yaml

### PR DESCRIPTION
**Pull request by Adamusen**
Set the initial optimizer.max_lr value to be zero for each parameter group. Removed the "baked in" 0.8 optimizer momentum values, it will be initialized with the one provided in the train config file instead.

Fixes #122

### CODE REVIEW

**Comment by Adamusen** [comment]
Alright :)

One additional note regarding the learning rate scheduling: With the current implementation the lightning module is unable to restore the learning rate if one tries to continue an interrupted training by providing the checkpoint path for `trainer.fit(model, ckpt_path=ckpt_path)` in lazy.py :) (otherwise everything else is loaded properly)

**Comment by Adamusen** [comment]
Closing this pull request, as you are working on this part of the code yourself anyway :)
